frontend/apps/marketing/src/app/[brand]/[locale]/layout.tsx

```diff
@@ -37,6 +38,15 @@ export default async function Layout({
   const localeConfig = SUPPORTED_LOCALES_MAP.get(locale);
   const theme = getMuiTheme(brand);
   const isDraftModeEnabled = (await draftMode()).isEnabled;
+  // Get Header component based on brand
+  const getHeader = () => {
+    switch (brand) {
+      case Brand.CS_FOR_ALL:
+        return <HeaderCSforAll />;
+      case Brand.CODE_DOT_ORG:
+        return <HeaderCorporateSite />;
+    }
+  };
```

Comment:
@stephenliang where is the best place to put this?

Reply:
I'd put these in `@/components/Header`

Reply:
Done! https://github.com/code-dot-org/code-dot-org/pull/67457/commits/6aee1329d8ad39dfe2257294d26fe3db68a85ec8

Reply:
Yep I'm going to do that once I refactor the Code.org Footer, see [this ticket](https://codedotorg.atlassian.net/browse/CMS-964).